DESCRIPTION

```diff
@@ -63,7 +63,7 @@ Imports:
     MsCoreUtils (>= 1.15.5),
     MsFeatures,
     MsExperiment (>= 1.5.4),
-    Spectra (>= 1.13.7),
+    Spectra (>= 1.15.7),
```

Comment:
I Guess needed because only the newer has `estimatePrecursorIntensity`

Reply:
Yes, because for `MsExperiment`/`XcmsExperiment` we're using the `estimatePrecursorIntensity,Spectra` method, while for the older `XCMSnExp` object we use a different one.